src/applications/check-in/locales/en/translation.json

```diff
@@ -153,5 +153,6 @@
   "date-of-birth": "Date of birth",
   "your-date-of-birth-can-not-be-in-the-future": "Your date of birth can not be in the future",
   "please-provide-a-response": "Please provide a response",
-  "in-en": "in english"
+  "in-en": "in english",
+  "sorry-we-cant-complete-pre-check-in": "Sorry, we can’t complete pre-check-in"
```

Comment:
This key was also part of my PR and will probably be duplicated - might want to merge in the latest master to get that sorted.

Reply:
nice catch!